analyzers/packaging/SonarAnalyzer.CSharp.nuspec

```diff
@@ -12,7 +12,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Roslyn analyzers that spot Bugs, Vulnerabilities and Code Smells in your code. For an even better overall experience, you can use SonarLint for Visual Studio, which is a free extension that can be used standalone or with SonarQube and/or SonarCloud.</summary>
     <description>Roslyn analyzers that spot Bugs, Vulnerabilities and Code Smells in your code. For an even better overall experience, you can use SonarLint for Visual Studio, which is a free extension (https://www.sonarlint.org/visualstudio/) that can be used standalone or with SonarQube (https://www.sonarqube.org/) and/or SonarCloud (https://sonarcloud.io/).</description>
-    <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.39.0.0</releaseNotes>
+    <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.40.0.0</releaseNotes>
```

Comment:
When does this get updated to the link with the right minor version? Like https://github.com/SonarSource/sonar-dotnet/releases/tag/8.39.0.47922

Reply:
At the beginning of the CI run - this ugly part is also executed by the `set-version.ps1` script to update the .NET part of the project. It also updates the shared file with version.
 
https://github.com/SonarSource/sonar-dotnet/blob/de8da8f78193e30579cf07cae0f5cae5113d33a9/azure-pipelines.yml#L61-L66